sorc/chgres_cube.fd/search_util.F90

```diff
@@ -215,27 +215,47 @@ subroutine search (field, mask, idim, jdim, tile, field_num, latitude, terrain_l
 
  end subroutine search
 
-!> Set sst values based on latitude.
+!> Set default sst values based on latitude.
```

Comment:
When using SST as an acronym, please capitalize it for clarity.

Also, the first time using an acronym in a documentation block (i.e. the first sentence in this case), define it:

Set default Sea Surface Temperatures (SST) values based on latitude.